riocli/configtree/revision.py

```diff
@@ -40,10 +41,10 @@ def __init__(self, tree_name: str,
                  commit: bool = False,
                  force_new: bool = False,
                  spinner: Optional[Yaspin] = None,
-                 with_project: bool = True):
+                 with_org: bool = True):
```

Comment:
todo: Please update on line 55 the usage of with_project

Reply:
I'll revert this back to `with_project` since within the context of Revision it makes sense. The initialization of an instance could have this instead `, with_project=(not with_org)`.

However, there's no usage that explicitly passes it right now.